community.md

```diff
@@ -3,6 +3,82 @@ layout: sidebar
 title: Community
 permalink: /community/
 published: false
+tags: label distribution install remove
+contributors: caitmuenster muffinresearch bqbn
+author: caitmuenster
+date: 2019-06-24 09:00:00
 ---
 
-# Community
+<!-- Overview Page Hero Banner -->
+
+<section class="overview-hero" style="background-image: url({% asset "community-overview-hero-bg.jpg" @optim @path %});">
```

Comment:
Could you describe the strange wrapping you're seeing? Sorry I'm not clear on what it is.

Reply:
Just that the line is wrapped rather than being on a single line. Maybe prettier is doing this - if it is that could be changed via the configuration.

Reply:
Oh, that's strange. It's not actually wrapping for me locally - even after prettier.

Reply:
I think we might have wires crossed, I meant the line of code not when rendered.

Reply:
This is what I see: 
<img width="931" alt="Screen Shot 2019-06-24 at 9 26 45 AM" src="https://user-images.githubusercontent.com/5910686/60035673-429c1f00-9662-11e9-858a-6b3365e66a55.png">

Reply:
Ah nevermind then - this is fine.